src/engine/graphic/GRStaffManager.cpp

```diff
@@ -122,6 +122,9 @@ GRStaffManager::GRStaffManager(GRMusic * p_grmusic, ARPageFormat * inPageFormat,
 		// Apply default layout settings
 		GuidoGetDefaultLayoutSettings (&this->settings);
 	}
+    
```

Comment:
I copied the map to be consistent with settings which are copied too (whereas it's duplicated, if I'm not wrong).